src/server/index.test.jsx

```diff
@@ -98,6 +102,15 @@ describe('Server', () => {
           expect(statusCode).toEqual(404);
         });
       });
+
+      describe('Trailing slash redirects', () => {
```

Comment:
Moved from [here](https://github.com/bbc/simorgh/pull/3326/files#diff-9d7ac6551ba249f8b551e1ca9306caa4L118) because it is an article URL not a front page one and seemingly was under the wrong describe block.